src/lib/record-filter.js

```diff
@@ -58,9 +57,8 @@ export function getFieldFilter (name, kind, query, operator = '') {
 
   // Since userID and recordID must be numbers, we check if query is number to avoid wrong queries
   if (['DateTime'].includes(kind)) {
-    const date = moment(query)
-    if (date.isValid()) {
-      return `DATE(${name}) ${operator || '='} DATE('${date.format('YYYY-MM-DD')}')`
+    if (query.isValid()) {
+      return `DATE(${name}) ${operator || '='} DATE('${formatDate(query)}')`
```

Comment:
Leave this one untouched; we need to pass raw-date in simple format to the backend here.